src/main/kotlin/uk/gov/justice/digital/hmpps/oneloginuserregistry/dto/orchestration/PrisonerBasicInfoDto.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.oneloginuserregistry.dto.orchestration
+
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDate
+
+data class PrisonerBasicInfoDto(
+  @Schema(required = true, description = "Prisoner Number", example = "A1234AA")
+  val prisonerNumber: String,
```

Comment:
nice to have required at end, like we have done in other projects

Reply:
Updated